mongodb-panache-quickstart/src/main/java/org/acme/mongodb/panache/Person.java

```diff
@@ -10,23 +10,15 @@
 @MongoEntity(collection="ThePerson")
 public class Person extends PanacheMongoEntity {
     public String name;
-
     // will be persisted as a 'birth' field in MongoDB
     @BsonProperty("birth")
     public LocalDate birthDate;
-
     public Status status;
 
-    // return name as uppercase in the model
     public String getName(){
         return name.toUpperCase();
     }
 
-    // store all names in lowercase in the DB
-    public void setName(String name){
-        this.name = name.toLowerCase();
-    }
```

Comment:
This is included in the doc so it's a bit annoying to change that: https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/mongodb-panache.adoc#defining-your-entity

At least it should be a coordinated change.